public/js/views/animations.coffee

```diff
@@ -0,0 +1,100 @@
+BaseView = require "./baseView.coffee"
```

Comment:
Can we put this file in the helpers folder? Since this is technically not a view.